app/router.js

```diff
@@ -28,7 +28,9 @@ Router.map(function () {
         this.route(
           'minister',
           { path: '/bedienaar/:ministerId/' },
-          function () {}
+          function () {
+            this.route('edit');
```

Comment:
I just noticed that the mandatory edit route is in the administrative unit module instead of the person one ([for example](https://dev.organisaties.abb.lblod.info/bestuurseenheden/a7164f51e0e68820b9d2d4d53aaf48f9/bestuursorganen/44883739b088afd2f1360f8470f30233/mandataris/ded98420a8e608689cb9492ced914342)). So this PR isn't consistent with that. I think it makes sense to move that one to the person module as well, but it requires more changes:

The problem with the current implementation in this PR is that the user always returns to the person position details page, even if they come from the administrative unit. This is weird, especially if the user presses cancel:

1. Go to administrative units minister page
2. Click bewerk
3. Click cancel or save
4. Notice that the user is redirected to the person position details page

If we keep it in the person module, we probably need to implement some redirect logic similar to the person creation page.

Reply:
I discussed with anita about that and given the "confusion" related to the path to reach the ministers and the ground inconsistency with reference to mandataris we should to put them here for now.

Reply:
I'm not sure what you mean with 'put them there for now'. But like I said, if we keep it in the person module I would add the same redirect system that we use when creating a person. Unless you get confirmation from Yassin or the Design team that it isn't needed.

Reply:
Yes the idea is to discuss with Yassin about that after the standup :).

Reply:
Yessin confirm that we should redirect to the ministers list in the administrative unit.  I should use something like that, right?

```
    if (this.redirectUrl) {
      // When passing a url the query params are ignored so we add the person id manually for now
      this.router.transitionTo(`${this.redirectUrl}?personId=${personId}`);
    } else {
      this.router.transitionTo('people.person', personId);
    }
```

Reply:
Yes, the personId part isn't needed here though.

So the general idea is to add a `redirectUrl` to this edit page, which is reset when leaving the page. Other pages can then set this query parameter when navigating to the edit page.

The edit page itself can then use this query parameter if it is set, or use the person route if it is not.

So it's very similar to the person creation page setup, but without the need to pass an id when redirecting back.

Let me know if you need more information.

Reply:
I did it but only for the "Bewerk" link in the ministers list page. Otherwise if the user first visit the position details page and than try to modify it through the "Bewerk" link on the top-righ, he'll will be redirected again to the details page where he was before. It makes more sense like that, do you agree?